pkg/infra/tracing/tracing.go

```diff
@@ -267,10 +267,14 @@ func splitCustomAttribs(s string) ([]attribute.KeyValue, error) {
 func (ots *Opentelemetry) initJaegerTracerProvider() (*tracesdk.TracerProvider, error) {
 	var ep jaeger.EndpointOption
 	// Create the Jaeger exporter: address can be either agent address (host:port) or collector URL
-	if host, port, err := net.SplitHostPort(ots.Address); err == nil {
-		ep = jaeger.WithAgentEndpoint(jaeger.WithAgentHost(host), jaeger.WithAgentPort(port), jaeger.WithMaxPacketSize(64000))
-	} else {
+	if strings.HasPrefix(ots.Address, "http://") || strings.HasPrefix(ots.Address, "https://") {
```

Comment:
nit; wouldn't it be better to use url.Parse?

Reply:
It might be tempting to use proper parsing functions from the `net` package, however:
* Historically, we have two valid forms of syntax: `host:port` for UDP and `https://example.com/` for collector URL.
* Using `url.Parse` would succeed on both, returning `host` as a schema and `port` as host in the first case.
* Similarly, `net.SplitHostPort` would succeed on both, returning `https` as a host and `//example.com/` as a very weird port syntax.

Which is why the only relatively simple way to tell one from the other is to use string prefixes, knowing that the only valid schema for URLs is http(s).